site/_posts/2017-09-29-miqldap-to-sssd.md

```diff
@@ -342,4 +343,42 @@ This document has diagnostics tips, including:
   - How to use **dbus-send**, with usage examples
   - Adding **debug_level=9** to the sssd.conf to produce more sssd logging information
 
+**Edge Cases**
+=====================================================================
+
+The *miqldap_to_sssd* conversion tool attempts to make assumptions about the initial configuration in order to produce a working SSSD configuration.
+Because there are so many different possibilities these authentication configurations assumptions may not be completely accurate, resulting in
```

Comment:
I think "configurations" should be "configuration" opn this line

Reply:
Also, I think there should be a comma after "possibilities" on this line

Reply:
@gtanzillo Thank you for looking this over. I'm making the suggested changes now.